downstream/modules/platform/con-ocp-supported-install.adoc

```diff
@@ -7,29 +7,29 @@ You can use the OperatorHub on the {OCP} web console to install {OperatorPlatfor
 
 Alternatively, you can install {OperatorPlatform} from the {OCPShort} command-line interface (CLI), `oc`.
 
-After you have installed {OperatorPlatform} you must create an *AnsibleAutomationPlatform* custom resource (CR). This allows you to manage {PlatformNameShort} components from a single unified interface known as the platform gateway. As of version 2.5, you must create an {PlatformNameShort} CR, even if you have an existing automation controller,  automation hub, or Event-driven Ansible (EDA), components.
+After you have installed {OperatorPlatform} you must create an *AnsibleAutomationPlatform* custom resource (CR). This allows you to manage {PlatformNameShort} components from a single unified interface known as the platform gateway. As of version 2.5, you must create an {PlatformNameShort} CR, even if you have an existing {ControllerName},  {HubName}, or {{EDAName}}, components.
```

Comment:
Looks like `{{EDAName}}` has an extra set of curly brackets

Reply:
Fixed!